highfidelity/hifi
# Open Asset Browser with Button from Edit.js Toolbar

This PR makes it so that you can open the Asset Browser from the Edit.js toolbar.

To test:
1. Click 'Edit'
2. Observe the new "Assets" button in the leftmost slot
3. When you click on it, the Asset Browser should open.
